widgets/src/api/makeRequest.js

```diff
@@ -51,5 +51,9 @@ export default async ({
 
   const responseHeaders = response.headers.get('Content-Type');
 
-  return responseHeaders && responseHeaders.includes('xml') ? response.text() : response.json();
+  return responseHeaders && responseHeaders.includes('xml')
+    ? response.text()
+    : responseHeaders && responseHeaders.includes('octet')
+    ? response
+    : response.json();
```

Comment:
```suggestion
  if (responseHeaders && responseHeaders.includes('xml')) {
    return response.text();
  }
  if (responseHeaders && responseHeaders.includes('octet')) {
    return response;
  }
  return response.json();
```

Reply:
changed the logic to be more readable, thanks!